.github/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## Release 0.4.0 (development release)
+
+### Contributors
+
+This release contains contributions from (in alphabetical order):
+
 ## Release 0.3.8
```

Comment:
We need to update the current release too. I'm surprised the GA workflow didn't take care of that already. 🤔 
```suggestion
## Release 0.3.8 (current release)
```